### PR TITLE
Fixed section after element. Fixed footer

### PR DESCRIPTION
**Pull request by kharkov-timur**

### CODE REVIEW

**Comment by mytrichq** [comment]
Fixed this issues in my pr

**Review by liudmylasovetovs — APPROVED** [review]

File: src/css/footer.css [hunk 1]
```diff
@@ -5,8 +5,8 @@
   gap: 40px;
 }
 
-.footer {
-  padding: 40px 0;
+footer {
```
Comment (mytrichq) [inline src/css/footer.css:8]: use class instead of tag selector

File: src/css/footer.css [hunk 2]
```diff
@@ -175,7 +175,7 @@
 
 @media screen and (min-width: 768px) {
   footer {
```
Comment (mytrichq) [inline src/css/footer.css:177]: same as above

File: src/css/footer.css [hunk 3]
```diff
@@ -206,7 +206,7 @@
 
 @media screen and (min-width: 1280px) {
   footer {
```
Comment (mytrichq) [inline src/css/footer.css:208]: same as above

**Pull request merged**